content/docs/nextjs/github-public-repo.md

```diff
@@ -183,21 +226,19 @@ export default function Authorizing() {
 
 ## Setup the GitHub OAuth App
 
-In GitHub, within your Account Settings, click <a href="https://github.com/settings/developers" target="_blank">Oauth Apps</a> under Developer Settings. Go ahead and create a "New Oauth App".
-
-For the **Authorization callback URL**, enter the url for the "/github/authorizing" page that you created above (e.g https://your-url/github/authorizing). Fill out the other fields with your custom values.
+Now we need to setup an OAuth App in Github to allow for authorization. In GitHub, within your Account Settings, click <a href="https://github.com/settings/developers" target="_blank">Oauth Apps</a> under Developer Settings. Go ahead and create a "New Oauth App".
 
-> _Note:_ If you are **testing your app locally**, you may need a separate development GitHub app (with a localhost redirect), and a production GitHub app.
+Since you are **testing your app locally**, eventually you'll need to create separate OAuth Apps: one for development, with a localhost redirect, and a production GitHub app whose URLs will connect to the 'live' domain.
 
-Next we will add the **Client ID** and **Client Secret** to your environment variables.
+For now, fill in http://localhost:3000 for the _Homepage Url_. And with the **Authorization callback URL**, enter the url for the "/github/authorizing" page that you created above (e.g http://localhost:3000/github/authorizing).
 
-<!-- TODO: add ## Developing Locally section at this point? to explain where the content is sourcing from, etc. -->
+After creating the app, you should see a page with information such as **Client ID** and **Client Secret**. Next, we'll add those as environment variables to the project to connect this App to the Tina-GitHub helpers.
 
 ### Setting Environment Variables
 
-This guide will reference [environment variables](https://nextjs.org/docs/api-reference/next.config.js/environment-variables) — these are sensitive values specific to your project. The Tina helpers will use these value to talk to your repository, enabling auth and data fetching via GitHub.
+[Environment variables](https://nextjs.org/docs/api-reference/next.config.js/environment-variables) are sensitive values specific to your project. The Tina-GitHub helpers will use these value to talk to your repository, enabling auth and data fetching via GitHub.
```

Comment:
```suggestion
[Environment variables](https://nextjs.org/docs/api-reference/next.config.js/environment-variables) are sensitive values specific to your project. The Tina-GitHub helpers will use these values to talk to your repository, enabling auth and data fetching via GitHub.
```
I'm glad I can be making such hard-hitting contributions at my first week on the job.